control/server.py

```diff
@@ -243,3 +265,12 @@ def _ping(self):
         except Exception as ex:
             self.logger.error(f"spdk_get_version failed with: \n {ex}")
             return False
+
+    def _update_state(self):
+        """Periodically check for gateway state updates."""
+        update_interval = self.config.getint("gateway", "state_update_interval_sec")
```

Comment:
We should define a range of valid values, e.g. what happens if someone specifies a negative value? Also should we allow to set `0` to indicate no periodic update? (Put a big warning log message in this case).

Reply:
We discussed this today in the community call and decided to not allow to disable the polling since it is always required to keep the state consistent.